swupd/swupd_test.go

```diff
@@ -180,10 +181,13 @@ func TestProcess(t *testing.T) {
 	if mp.output != "" {
 		t.Fatal("Message processed incorrectly. Expected: '', Actual:", mp.output)
 	}
+	//nolint: lll // WONTFIX
```

Comment:
What is the scope of this //nolint?

Reply:
scope is limited to variable following //nolint: jsonMSG  and only for lll